Formula/rubberband.rb

```diff
@@ -18,11 +18,11 @@ class Rubberband < Formula
   depends_on "pkg-config" => :build
   depends_on "libsamplerate"
   depends_on "libsndfile"
-  unless OS.mac?
+  on_macos do
```

Comment:
```suggestion
  on_linux do
```

because I can't `unless` 😭

Reply:
Oh duh 🤦